translations/en_us.json

```
@@ -59,7 +59,7 @@
       "removeOption": "Remove search option",
       "addOption": "Add search option",
       "resultsHeader": "Search results",
-      "noResult": "No results",
+      "noResult": "Gestis returned no results.\nPlease try again with synonyms or partial words.",
```

```suggestion
      "noResult": "%Provider returned no results.\nPlease try again with synonyms or partial words.",
```
@Epsilon02 do transloco support variables?

---

Like I said in the description, let's use the quick fix and find a better solution later, because different providers could use different help messages

---

To answer @6543 question anyway.
Yes it is possible to use variables.
https://ngneat.github.io/transloco/docs/translation-in-the-template

```html
     <p>{{ t('noResult', { provider: provider }) }}</p>
```
```json
"noResult": "{{provider}} returned no results.\nPlease try again with synonyms or partial words."
```

---

@crapStone is there a follow up issue for this quick fix?

---

no, I won't close the current one

---

done

---

we should fix all crowdin translations as fast as possible

---

> no, I won't close the current one

now I close it